ossdbtoolsservice/connection/connection_service.py

```diff
@@ -203,17 +203,7 @@ def handle_list_databases(self, request_context: RequestContext, params: ListDat
         try:
             connection = self.get_connection(params.owner_uri, ConnectionType.DEFAULT, request_context)
         except ValueError as err:
-            request_context.send_notification(
-                method = TELEMETRY_NOTIFICATION,
-                params = TelemetryParams(
-                    TELEMETRY_ERROR_EVENT,
-                    {
-                        'view' : 'Connection',
-                        'name': 'List Databases Connection Value Error',
-                        'errorCode': str(OssdbErrorConstants.LIST_DATABASE_GET_CONNECTION_VALUE_ERROR)
-                    }
-                )
-            )
+            send_error_telemetry_notification(request_context, OssdbErrorConstants.CONNECTION, OssdbErrorConstants.LIST_DATABASES_CONNECTION_VALUE_ERROR, str(OssdbErrorConstants.LIST_DATABASE_GET_CONNECTION_VALUE_ERROR))
```

Comment:
instead of sending error code by type casting as str in every call.
Pass it as int and covert to str in the send_error_telemetry_notification.

Reply:
done